mbed/mbed.py

```diff
@@ -3356,9 +3356,10 @@ def main():
         very_verbose = pargs.very_verbose
         verbose = very_verbose or pargs.verbose
         pathtype = Repo.pathtype(cwd_root)
-        action('Working path \"%s\" (%s)' % (cwd_root, pathtype))
-        if pathtype != "program":
-            action('Program path \"%s\"' % Program(cwd_root).path)
+        if not sys.argv[1].lower() in ["config", "cfg", "conf"]: # `mbed config` should be parse-friendly
```

Comment:
Perhaps this should be a global, so that if I add another alias for config I don't have to add it into two places. You could also remove the comment at that point.

Reply:
Not sure what's the proposal? Modify the subcommand routine so for every command there's a list of aliases?

Reply:
make `["config", "cfg", "conf"]` a global variable.